packages/gafl-webapp-service/src/locales/cy.json

```diff
@@ -529,14 +531,27 @@
   "order_complete_when_fishing_salmon_link": "submit a catch return (opens in new tab) ",
   "order_complete_when_fishing_salmon_para_part2": "of yearly salmon and sea trout fishing activity in England and Wales. A catch return must be submitted even if no fish are caught or no fishing activity took place.",
   "order_complete_help_us": "Help us improve this service",
-  "order_complete_tell_us_link": "Tell us what you think of this service",
   "order_complete_tell_us_time": " (takes 30 seconds).",
-  "order_complete_title_application": "Application complete",
-  "order_complete_title_payment": "Payment complete",
   "order_complete_income_spend": "How we spend fishing licence income",
   "order_complete_annual_report_part1": "Read the latest ",
   "order_complete_annual_report_link": "fisheries annual report (opens in new tab)",
   "order_complete_annual_report_part2": " to find out how we spend the money from fishing licence sales.",
+  "order_complete_conf_1": "Rhif y drwydded yw ",
+  "order_complete_conf_2": "Trwydded i ddechrau am ",
+  "order_complete_conf_info_1": "Os ydych wedi dewis yr opsiwn di-bapur, bydd y drwydded yn cael ei hanfon ar ffurf neges destun neu e-bost. Os ydych wedi dewis yr opsiwn cerdyn, bydd y drwydded yn cyrraedd drwy’r post ymhen 15 diwrnod gwaith.",
+  "order_complete_conf_info_2": "Gall deiliaid trwydded bysgota o ddyddiad ac amser dechrau’r drwydded ac nid oes rhaid iddynt aros i’r cerdyn gyrraedd.",
+  "order_complete_conf_info_3": "Mae’n rhaid darparu rhif y drwydded i swyddog gorfodi pe byddai angen.",
+  "order_complete_subheading_rcr": "Cyflwynwch ddatganiad o’ch dalfa flynyddol",
+  "order_complete_tell_us_link": " (bydd hyn yn cymryd 30 eiliad).",
+  "order_complete_tell_us": "Rhowch eich barn ar y gwasanaeth hwn",
```

Comment:
This line has no equivalent in the en file and does not seem to be used in the Nunjucks template either - not sure where it came from